react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      view: 'none'
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(e) {
+    const id = e.target.getAttribute('id');
+    if (this.state.view === id) {
+      this.setState({
+        view: 'none'
+      });
+    } else {
+      this.setState({
+        view: id
+      });
+    }
+  }
+
+  showView() {
+    let description;
```

Comment:
you can move this variable declaration into your `.map()` callback. Also, I recommend a more accurate name, like `descriptionClass`.